aspnet/signalr/overview/guide-to-the-api/hubs-api-guide-server.md

```diff
@@ -298,6 +298,9 @@ To call client methods from the server, use the `Clients` property in a method i
 
 [!code-csharp[Main](hubs-api-guide-server/samples/sample23.cs?highlight=5)]
 
+> [!NOTE]
+> Invoking a client method is an asynchronous operation and returns a `Task`. Use `await` to ensure you wait for the message to be sent. If an error occurs while sending the message, `await` will also allow you to catch that exception using a try-catch block.
```

Comment:
You could maybe link to the appropriate API docs here. Just a thought.

Reply:
Yeah, I actually did some searching to try and find appropriate docs on observing exceptions and didn't really like what I found.

Reply:
https://docs.microsoft.com/en-us/dotnet/standard/parallel-programming/exception-handling-task-parallel-library - Doesn't really cover `async`/`await`

https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.taskscheduler.unobservedtaskexception?view=netframework-4.7.2 - Very light on narrative descriptions

https://blogs.msdn.microsoft.com/pfxteam/2011/09/28/task-exception-handling-in-net-4-5/ - I trust the source a lot but it's not really a "doc"

Reply:
> Yeah, I actually did some searching to try and find appropriate docs on observing exceptions and didn't really like what I found.

@BillWagner 	can you suggest a link?

Invoking a client method is an asynchronous operation and returns a `Task`. Use `await` :
* To ensure the message is sent without error. 
* To enable catching and handling errors.

I think most folks know  you'd need a try/catch.